### PR TITLE
update createrepo_c to 0.17.7

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: packages/createrepo_c/createrepo_c.spec [hunk 1]
```diff
@@ -42,8 +42,8 @@
 
 Summary:        Creates a common metadata repository
 Name:           createrepo_c
-Version:        0.17.6
-Release:        4%{?dist}
+Version:        0.17.7
+Release:        1%{?dist}
```
Comment (evgeni) [inline packages/createrepo_c/createrepo_c.spec:46]: CentOS Stream 8 contains `0.17.7-3.el8`, so we'd need to bump this to 4 to be actually newer than them.

Which raises the question, whether this is a smart move, or not…

Comment (evgeni) [reply packages/createrepo_c/createrepo_c.spec:46]: context, upgrades on c8s fail with:

```
[2021-12-15T00:02:18.370Z] TASK [update_os_packages : RedHat | Update packages] ***************************
[2021-12-15T00:02:18.370Z] [0;31mfatal: [pipe-up-katello-nightly-centos8-stream]: FAILED! => changed=false [0m
[2021-12-15T00:02:18.370Z] [0;31m  failures: [][0m
[2021-12-15T00:02:18.370Z] [0;31m  msg: |-[0m
[2021-12-15T00:02:18.370Z] [0;31m    Depsolve Error occured:[0m
[2021-12-15T00:02:18.370Z] [0;31m     Problem 1: package python38-createrepo_c-0.17.6-4.el8.x86_64 requires createrepo_c-libs = 0.17.6-4.el8, but none of the providers can be installed[0m
[2021-12-15T00:02:18.370Z] [0;31m      - package python38-pulp-rpm-3.16.1-1.el8.noarch requires python38-createrepo_c >= 0.17.6, but none of the providers can be installed[0m
[2021-12-15T00:02:18.370Z] [0;31m      - cannot install both createrepo_c-libs-0.17.6-4.el8.x86_64 and createrepo_c-libs-0.17.7-3.el8.x86_64[0m
[2021-12-15T00:02:18.370Z] [0;31m      - cannot install the best update candidate for package python3-pulp-rpm-3.14.8-1.el8.noarch[0m
[2021-12-15T00:02:18.370Z] [0;31m      - cannot install the best update candidate for package createrepo_c-libs-0.17.7-3.el8.x86_64[0m
[2021-12-15T00:02:18.370Z] [0;31m     Problem 2: problem with installed package createrepo_c-libs-0.17.7-3.el8.x86_64[0m
[2021-12-15T00:02:18.370Z] [0;31m      - cannot install both createrepo_c-libs-0.17.6-4.el8.x86_64 and createrepo_c-libs-0.17.7-3.el8.x86_64[0m
[2021-12-15T00:02:18.370Z] [0;31m      - package python38-createrepo_c-0.17.6-4.el8.x86_64 requires createrepo_c-libs = 0.17.6-4.el8, but none of the providers can be installed[0m
[2021-12-15T00:02:18.370Z] [0;31m      - package python38-pulp-rpm-3.16.1-1.el8.noarch requires python38-createrepo_c >= 0.17.6, but none of the providers can be installed[0m
[2021-12-15T00:02:18.370Z] [0;31m      - problem with installed package python3-pulp-rpm-3.14.8-1.el8.noarch[0m
[2021-12-15T00:02:18.370Z] [0;31m      - package python3-pulp-rpm-3.14.8-1.el8.noarch requires python3-django-readonly-field, but none of the providers can be installed[0m
[2021-12-15T00:02:18.370Z] [0;31m      - package python38-django-readonly-field-1.0.5-3.el8.noarch obsoletes python3-django-readonly-field < 1.0.5-3.el8 provided by python3-django-readonly-field-1.0.5-1.el8.noarch[0m
[2021-12-15T00:02:18.370Z] [0;31m      - cannot install the best update candidate for package python3-django-readonly-field-1.0.5-1.el8.noarch[0m
[2021-12-15T00:02:18.370Z] [0;31m  rc: 1[0m
[2021-12-15T00:02:18.370Z] [0;31m  results: [][0m
```

Comment (ehelms) [reply packages/createrepo_c/createrepo_c.spec:46]: Do we need to be newer than them? Could we do a 3.1 or something equivalent to be newer if we need to be but allow any further releases into stream to take precedence?

I think the answer is no, but given what we have learned recently - would Pulpcore having modular metadata help at all?

Comment (evgeni) [reply packages/createrepo_c/createrepo_c.spec:46]: We need a build that builds both 3.6 and 3.8 python, or CentOS needs to provide such a build.
As long as they do not, we need a matching set of createrepo_c, createrepo_c-libs, python3-createrepo_c and python38-createrepo_c

and no, modular metadata wouldn't help here.

**Review by Odilhao — APPROVED** [review]

**Pull request merged**